test/Tester/SerializationTests/SerializationTests.ImmutableCollections.cs

```diff
@@ -19,7 +19,8 @@ void RoundTripCollectionSerializationTest<T>(IEnumerable<T> input)
             var output = SerializationManager.RoundTripSerializationForTesting(input);
             Assert.Equal<T>(input,output);
         }
-
+#if !NETSTANDARD_TODO
+    //blocked by Cannot use binary formatter as fallback serializer while running on .Net Core
```

Comment:
You are partially right. in InitializeForTesting method, it's trying to GetFallbackSerializer by default. That failed a bunch of tests. I'm gonna mark that time using TODO. and remove others.